MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+include README.rst
+include LICENSE
+recursive-include allauth_2fa/templates *
```

Comment:
When I installed the package from the file system, the templates didn't come along. It seems that the default behavior is to just include `.py` files. I won't pretend to be an expert on Python packaging, but I've had this problem before and this seems to get the files included in the package.

The last line is the more important one. The others are nice extras. Since this isn't actually related to the routing, I can also split it into its own PR if you'd prefer.